src/tdyio.c

```diff
@@ -69,6 +71,7 @@ PetscErrorCode TdyIOInitializeExodus(char *ofilename, char *zonalVarNames[], DM
   PetscErrorCode ierr;
```

Comment:
Let's move all local variable definitions inside `ifdef` to avoid warnings

Reply:
Done (in all locations)